### PR TITLE
Libredwg

### PR DESCRIPTION
**Pull request by looooo**
continued from here:
https://github.com/conda-forge/staged-recipes/pull/8055

Checklist

- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vend other packages
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by looooo** [comment]
@scopatz using the source as compressed file from a release didn't work so far. I guess some info from for is needed to computed the internal version.

**Comment by scopatz** [comment]
That seems like a bug in their build process then.  Please use a patch rather than building from git. Thanks!

**Comment by looooo** [comment]
> That seems like a bug in their build process then. Please use a patch rather than building from git. Thanks!

Sry, we tried to solve this in the past, but no solution was found until now.

File: recipes/libredwg/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+test:
+  imports:
+    - LibreDWG
+
+about:
+  home: https://www.gnu.org/software/libredwg/
+  doc_url: https://www.gnu.org/software/libredwg/
+  dev_url: http://savannah.gnu.org/projects/libredwg/
+  license: GPL3
```
Comment (chrisburr) [inline recipes/libredwg/meta.yaml]: SPDX identifiers are preferred:
```suggestion
  license: GPL-3.0-or-later
```

File: recipes/libredwg/build.sh [hunk 2]
```diff
@@ -0,0 +1,27 @@
+echo "####################################"
+echo $(ls -al)
+echo "####################################"
+
+# TODO: Fix this upstream
+mv git-version-gen build-aux/git-version-gen
```
Comment (chrisburr) [inline recipes/libredwg/build.sh]: Can you open an issue upstream to get this included in the release tarballs? I'm not sure why it's only in the repository.

File: recipes/libredwg/build.sh [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        ;;
+    MINGW*)
+        export PATH="$PREFIX/Library/bin:$BUILD_PREFIX/Library/bin:$RECIPE_DIR:$PATH"
+        export CFLAGS="-MD -I$PREFIX/Library/include -O2 -DM4RI_USE_DLL"
+        export LDFLAGS="$LDFLAGS -L$PREFIX/Library/lib"
+        ./configure --prefix="$PREFIX/Library" --libdir="$PREFIX/Library/lib" --disable-sse2
+        ;;
+esac
+
+make
```
Comment (chrisburr) [inline recipes/libredwg/build.sh]:
```suggestion
make -j${CPU_COUNT}
```

File: recipes/libredwg/meta.yaml (hunk 1 above)
Comment (looooo) [reply recipes/libredwg/meta.yaml]: :+1:

File: recipes/libredwg/build.sh (hunk 3 above)
Comment (looooo) [reply recipes/libredwg/build.sh]: :+1:

File: recipes/libredwg/build.sh (hunk 2 above)
Comment (looooo) [reply recipes/libredwg/build.sh]: @chrisburr thanks for having a look at this. I guess it's best to ping @rurban here.

**Comment by chrisburr** [comment]
Thanks for adding this @looooo!

**Pull request merged**